.pairs

```diff
@@ -1,17 +1,27 @@
 # .pairs - configuration for 'git pair'
 pairs:
   # <initials>: <Firstname> <Lastname>[; <email-id>]
-  jh: Jenny Heath; jheath
-  bg: Ben Golder; bgolder
-  td: Tom Dooner; tdooner
-  lk: Laura Kogler; lkogler
-  mtm: Molly T-M; mollyt
-  jg: Jonathan Greenberg; jgreenberg
-  bv: Ben Vandgrift; bvandgrift
-  dbd: Dimitri DeFigueiredo; ddefigueiredo
-  al: Asheesh Laroia; alaroia
-email:
-  prefix: pair
-  domain: codeforamerica.org
-  no_solo_prefix: true
+  jh: Jenny Heath
+  bg: Ben Golder
+  td: Tom Dooner
+  lk: Laura Kogler
+  mtm: Molly T-M
+  jg: Jonathan Greenberg
+  bv: Ben Vandgrift
```

Comment:
nonblocking but molly, jonathan, and ben vandgrift can all be removed from here if you want to make that change as well!

Reply:
Got it!